src/example-relay/flow-typed/globals.js

```diff
@@ -0,0 +1 @@
+declare var __DEV__: boolean;
```

Comment:
Why aren't these `flow-typed` files under `__github__` as well? (they are used in `src/example-relay/__github__/.flowconfig` for example so it would make sense)

Reply:
fixed